### PR TITLE
get_armor Int Casting Safeguard

### PR DESCRIPTION
**Pull request by Turtlelord26**
From Issue #64
Now finds blank armor stats and makes them '0' before they are casted to ints.

### CODE REVIEW

File: start.py [hunk 1]
```diff
@@ -137,6 +137,9 @@ def get_armor(session):
 	for type in arms:
 		toHide = []
 		for set in arms[type]:
+			for k, v in set.items():
+				if v = ''
```
Comment (bleehu) [inline start.py]: missing colon at end of conditional.

File: start.py [hunk 2]
```diff
@@ -137,6 +137,9 @@ def get_armor(session):
 	for type in arms:
 		toHide = []
 		for set in arms[type]:
+			for k, v in set.items():
+				if v = '':
```
Comment (bleehu) [inline start.py]: need == for comparison instead of = for assignment here

**Comment by bleehu** [comment]
ahah! I didn't know this; set is a keyword in python. It's the litteral type "set".

**Comment by bleehu** [comment]
I'll fix it from here; this is definitely my bug and you've done the main fix already. Can you tackle the How To General and How To Enemies pages?

**Comment by Turtlelord26** [comment]
On my list for after I finalize the space 2.0 test.

**Pull request merged**